23_1.cpp

```diff
@@ -0,0 +1,59 @@
+#include <functional>
+#include <iostream>
+#include <queue>
+#include <vector>
+
+struct ListNode {
+  int val;
+  ListNode* next;
+  explicit ListNode(int x) : val(x), next(nullptr) {}
+};
+
+struct IdWithValue {
+  int id_;
+  int value_;

```

Comment:
In this case (small struct with public fields only), variables should be named as `id`, `value` (without the tailing `_`) according to Google coding style.

Reply:
ok